heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java

```diff
@@ -278,8 +280,8 @@ public static boolean curlPackage(
 
     // using curl copy the url to the target file
     String cmd = String.format("curl %s -o %s", uri, destination);
-    int ret = runSyncProcess(isVerbose, isInheritIO,
-        splitTokens(cmd), new StringBuilder(), new StringBuilder(), parentDirectory);
+    int ret = runSyncProcess(isInheritIO,
+        splitTokens(cmd), new StringBuilder(), parentDirectory);
```

Comment:
here and below we're passing an unused StringBuilder. can we refactor to not require that?